variables.tf

```diff
@@ -608,3 +608,9 @@ variable "realtime_log_config_arn" {
   default     = null
   description = "The ARN of the real-time log configuration that is attached to this cache behavior"
 }
+
+variable "allow_ssl_requests_only" {
+  type        = bool
+  default     = true
```

Comment:
Can defaulting this to `true` break any current deployments ?

Reply:
it shouldn't...since this module is specifically to create the S3 bucket that is the origin for CloudFront, and CloudFront uses SSL by default.